types/jsreport-core/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "esnext"
+            "es2020"
```

Comment:
```json
"es2020"
```
for me it looks wrong here, IMO, the `jsreport-core` is using `vars`,
what are these features from es2020 in DT?

Reply:
Is this lib bug still impacting this definition? (sry, just read)

Reply:
having issues running tests locally so trying here

Reply:
It appears that esnext is working once again ;)